hugr-core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ workspace = true
 extension_inference = []
 declarative = ["serde_yaml"]
 model_unstable = ["hugr-model"]
+default = ["model_unstable"]
```

Comment:
At the point where we are making "unstable" the default I have to wonder if we still need a flag for the base/stable model ?

Reply:
This isn’t intentional, good for spotting it. I got to figure out how to tell the lsp to work without needing to make the feature default, because I keep forgetting to take it out…